docs/pages/docs/connectors/coinbaseWallet.en-US.mdx

```diff
@@ -38,6 +38,8 @@ const connector = new CoinbaseWalletConnector({
 })
 ```
 
+Note: Upon connection, the connector will attempt to connect to the first chain in the array unless otherwise specified by a [`chainId` config on useConnect](/docs/hooks/useConnect#chainid-optional).
```

Comment:
These docs need to be updated to mention that it attempts to reconnect to the last used chain.